regen-network/regen-web
# feat: switch wallet warning modal

## Description

Closes: regen-network/regen-web#2216

- display a warning modal whenever user tries to do an on-chain tx with a wallet different from the one linked to its active account

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



1. go to https://deploy-preview-2227--regen-marketplace.netlify.app/
2. log out (if already logged)
3. select a keplr address different from the one link to the account you're going to use in the next step
4. connect to an account with either passcode/google
5. try to actions that requires a wallet (eg: buy credits)
6. the modal should be displayed
7. change Kepler wallet to match the address associated with the active account
8. The waning modal should close and the action triggered should resume

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
